aspnetcore/data/ef-rp/concurrency.md

```diff
@@ -988,7 +988,7 @@ The following markup shows the updated page:
 
 ### Update the Edit page model
 
-Update *Pages\Departments\Edit.cshtml.cs* with the following code:
+Update  with the following code:
```

Comment:
Reference to the .cs file to update was removed.  Just checking in case that was not intentional.